### PR TITLE
Helm update

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

- This PR updates the Helm reference page with a new attribute for Palette 4.0+. 
  - [Palette](https://deploy-preview-1536--docs-spectrocloud.netlify.app/enterprise-version/helm-chart-install-reference#config)
  - [Vertex](https://deploy-preview-1536--docs-spectrocloud.netlify.app/vertex/install-palette-vertex/install-on-kubernetes/vertex-helm-ref#install-mode)
- Minor fixes to callout boxes not displaying in the [Airgap document](https://deploy-preview-1536--docs-spectrocloud.netlify.app/enterprise-version/air-gap-repo?install-method=palette-installer).
- Updated the backup instructions to reflect field feedback. 
CleanShot 2023-09-11 at 15 34 16
  - [Static Credentials](https://deploy-preview-1536--docs-spectrocloud.netlify.app/clusters/cluster-management/backup-restore/add-backup-location-static#prerequisites)
  - [Dynamic Credentials](https://deploy-preview-1536--docs-spectrocloud.netlify.app/clusters/cluster-management/backup-restore/add-backup-location-dynamic#prerequisites)

## Review Changes

💻 [Preview URL](https://deploy-preview-1536--docs-spectrocloud.netlify.app/enterprise-version/helm-chart-install-reference#config)

🎫 [DOC-869](https://spectrocloud.atlassian.net/browse/DOC-869)

### CODE REVIEW

File: docs/docs-content/enterprise-version/helm-chart-install-reference.md [hunk 1]
```diff
@@ -83,6 +83,20 @@ mongo:
...
 Review the following parameters to configure Palette for your environment. The `config` section contains the following subsections:
 
+
+#### Install Mode
+
+You can install Palette in connected or air-gapped mode. The table lists the parameters to configure the installation mode.
+
+| **Parameters** | **Description** | **Type** | **Default value** |
+| --- | --- | --- | --- |
+| `installMode` | Specifies the installation mode. Allowed values are `connected` or `airgap`. Set the value to `airgap` when conducting an install in an air gap environment. | String | `connected` |
```
Comment (ritawatson) [inline docs/docs-content/enterprise-version/helm-chart-install-reference.md:93]:
```suggestion
| `installMode` | Specifies the installation mode. Allowed values are `connected` or `airgap`. Set the value to `airgap` when installing in an air-gapped environment. | String | `connected` |
```

File: docs/docs-content/vertex/install-palette-vertex/install-on-kubernetes/vertex-helm-ref.md [hunk 2]
```diff
@@ -72,6 +72,20 @@ mongo:
...
 Review the following parameters to configure Palette VerteX for your environment. The `config` section contains the following subsections:
 
+
+#### Install Mode
+
+You can install Palette in connected or air-gapped mode. The table lists the parameters to configure the installation mode.
+
+| **Parameters** | **Description** | **Type** | **Default value** |
+| --- | --- | --- | --- |
+| `installMode` | Specifies the installation mode. Allowed values are `connected` or `airgap`. Set the value to `airgap` when conducting an install in an air gap environment. | String | `connected` |
```
Comment (ritawatson) [inline docs/docs-content/vertex/install-palette-vertex/install-on-kubernetes/vertex-helm-ref.md:82]:
```suggestion
| `installMode` | Specifies the installation mode. Allowed values are `connected` or `airgap`. Set the value to `airgap` when installing in an air-gapped environment. | String | `connected` |
```

**Review by ritawatson — APPROVED** [review]
Minor suggestions for polish. Otherwise LGTM.

File: docs/docs-content/clusters/cluster-management/backup-restore/add-backup-location-static.md [hunk 3]
```diff
@@ -97,6 +97,15 @@ The following sections provide detailed instructions. Select the environment whe
 	If you skip copying the secret access key, refer to the [Managing access keys for IAM users](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) guide to learn how to create a new access key.
 
 
+* If the S3 bucket is using a customer managed AWS Key Management Service (KMS) key for server-side encryption, ensure the Palette IAM user has the necessary permissions to access the KMS key. Otherwise, Palette will be unable to put objects in the S3 bucket and result in backup or restore failure. Check out the [Troubleshooting key access](https://docs.aws.amazon.com/kms/latest/developerguide/policy-evaluation.html) guide to learn more about common KMS issues.
+
+	:::tip
+
+	Use the IAM Policy Simulator to verify the IAM user has the necessary permissions to access a customer managed KMS key. Refer to the [Testing IAM policies with the IAM policy simulator](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_testing-policies.html) guide to learn more about the IAM Policy Simulator.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/clusters/cluster-management/backup-restore/add-backup-location-static.md:104]:
```suggestion
	Use the IAM Policy Simulator to verify the IAM role has the necessary permissions to access a customer managed KMS key. Refer to the [Testing IAM policies with the IAM policy simulator](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_testing-policies.html) guide to learn more.
```

**Pull request merged**